From 8b6c7500f016c3454dd0ba6dfb81a267e8bb7192 Mon Sep 17 00:00:00 2001
From: Sergey Larin <cerg2010cerg2010@mail.ru>
Date: Sat, 9 Feb 2019 18:44:17 +0300
Subject: [PATCH] ARM: dts: tegra20-glide: set panel reset GPIO pin to active
 low

This fixes display not working with new S6E63M0 panel driver.

Signed-off-by: Sergey Larin <cerg2010cerg2010@mail.ru>
---
 arch/arm/boot/dts/tegra20-glide.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/tegra20-glide.dts b/arch/arm/boot/dts/tegra20-glide.dts
index cb219d1b1198..87d737c62b0a 100644
--- a/arch/arm/boot/dts/tegra20-glide.dts
+++ b/arch/arm/boot/dts/tegra20-glide.dts
@@ -1230,7 +1230,7 @@
 			spi-cpha;
 			vdd3-supply = <&vlcd_1v8>;
 			vci-supply = <&vlcd_3v0>;
-			reset-gpios = <&gpio TEGRA_GPIO(C, 1) GPIO_ACTIVE_HIGH>;
+			reset-gpios = <&gpio TEGRA_GPIO(C, 1) GPIO_ACTIVE_LOW>;
 			panel-width-mm = <52>;
 			panel-height-mm = <87>;
 			/* 25 + 1 for regulator */
-- 
2.22.0

